tools/distrib/fix_build_deps.py

```diff
@@ -296,15 +296,22 @@ def grpc_cc_library(name,
         proto_hdr = '%s%s' % ((parsing_path + '/' if parsing_path else ''),
                               proto.replace('.proto', '.pb.h'))
         skip_headers[name].add(proto_hdr)
+
+    # Convert the source or header target to a relative path.
+    def get_filename(name, parsing_path):
```

Comment:
functions should be at global scope, not inlined

Reply:
Done